apis/scheduling/v1alpha1/types.go

```diff
@@ -37,7 +37,7 @@ type ElasticQuota struct {
 
 	// ElasticQuotaSpec defines the Min and Max for Quota.
 	// +optional
-	Spec ElasticQuotaSpec `json:"spec,omitempty" protobuf:"bytes,2,opt,name=spec"`
+	Spec ElasticQuotaSpec `json:"spec" protobuf:"bytes,2,opt,name=spec"`
```

Comment:
remove `omitempty` in a Spec field doesn't seem conventional. With the new CRD, how it looks if the user creates a bare ElastiQuota without specifying `Spec`?

Reply:
Thank you for your feedback.
I realized that I don't need to remove the omitempty tag. Therefore, I have decided to keep both `+optional` and `omitempty` tags.

> With the new CRD, how it looks if the user creates a bare ElastiQuota without specifying Spec?

If the user creates a bare ElasticQuota without specifying `Spec` like following Yaml, `max` and `min`are not created as default, which is not what I intended. Therefore, I have set up a default value for `Spec` as well.

Here is an example Yaml without specifying `Spec`.

```
apiVersion: scheduling.sigs.k8s.io/v1alpha1
kind: ElasticQuota
metadata:
  name: test
  namespace: test
```